Tests/EmbeddedScripts.Python.PythonNet.Tests/PythonNetRunnerTests.cs

```diff
@@ -151,12 +151,31 @@ public class A
         }
 
         [Fact]
-        public async void AddStructAsGlobals_Succeed()
+        public async Task AddStructAsGlobals_Succeed()
         {
             var runner = new PythonNetRunner();
             await runner
                 .Register(new A(), "a")
                 .RunAsync("a.X += 1;");
         }
+        
```

Comment:
Probably more tests can be added (similar to other engines for repl cases)

Reply:
Some of new tests are red at this moment